cmd/commands/connection/command.go

```diff
@@ -22,12 +22,12 @@ import (
 	"fmt"
 	"time"
 
+	"github.com/mysteriumnetwork/node/cmd/commands"
+
```

Comment:
Im gonna be a little annoying here but did you configure your linter correctly? This empty line should not be here.

Maybe go to settings and check if `goimports` is enabled?

Reply:
![image](https://user-images.githubusercontent.com/20511486/103284873-23afe300-49e5-11eb-8ad9-669ac76a6d7d.png)